preload/preloader/preloader.go

```diff
@@ -113,25 +110,25 @@ func precertSubmitter(ctx context.Context, addedCerts chan<- *preload.AddedCert,
 		copy(chain[1:], c.Chain)
 		sct, err := logClient.AddPreChain(ctx, chain)
 		if err != nil {
-			log.Printf("failed to add pre-chain with CN %s: %v", c.Precert.TBSCertificate.Subject.CommonName, err)
+			glog.Errorf("failed to add pre-chain with CN %s: %v", c.Precert.TBSCertificate.Subject.CommonName, err)
 			recordFailure(addedCerts, chain[0], err)
 			continue
 		}
 		recordSct(addedCerts, chain[0], sct)
-		if !*quiet {
-			log.Printf("Added precert chain for CN '%s', SCT: %s\n", c.Precert.TBSCertificate.Subject.CommonName, sct)
-		}
+		glog.V(2).Infof("Added precert chain for CN '%s', SCT: %s\n", c.Precert.TBSCertificate.Subject.CommonName, sct)
 	}
 }
 
 func main() {
 	flag.Parse()
+	glog.CopyStandardLogTo("WARNING")
+
 	var sctFileWriter io.Writer
 	var err error
 	if *sctInputFile != "" {
 		sctFile, err := os.Create(*sctInputFile)
 		if err != nil {
-			log.Fatalf("Failed to create SCT file: %v", err)
+			glog.Fatalf("Failed to create SCT file: %v", err)
```

Comment:
`glog` has `glog.Exitf`, which is preferable whenever the stack trace is uninteresting (which includes any pre-condition checking in `main`).

Reply:
Done.